app.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from dotenv import load_dotenv
+from flask import Flask, request, render_template, jsonify
+from flask_cors import CORS
+from libversion import VersionUtil
+import requests
+
+app = Flask(__name__)
+CORS(app)
+load_dotenv()
+
+
+@app.route('/')
+def home():
+    version_info = VersionUtil.VersionUtil.get_version()
+    return render_template("index.html", version=version_info)
+
+
+@app.route('/predict', methods=['POST'])
+def predict():
+    # Receive data from the front end
+    text = request.form['input_text']
+    model_choice = request.form['model_select']
+
+    model_service_url = os.getenv("MODEL_SERVICE_URL"),
+
+    payload = {
+        'input_url': text,
+        'model_name': model_choice
+    }
+
+    try:
+        response = requests.post(model_service_url, json=payload, timeout=10)  # timeout in seconds
```

Comment:
Nice that you managed to query the `model-service` through REST requests, thereby using a configurable URL!